services/watson-extension/src/watson_extension/clients/general/redhat_status.py

```diff
@@ -17,9 +18,10 @@ def __init__(
     ):
         super().__init__()
 
-    async def check_services_offline(self) -> ClientResponse:
+    async def check_services_offline(self) -> Optional[Dict]:
+        result = None
         try:
-            async with ClientSession() as session:
+            async with ClientSession(trust_env=True) as session:
```

Comment:
Does this skips the cert checks?

Reply:
From their [docs](https://docs.aiohttp.org/en/stable/client_reference.html#:~:text=in%20version%202.3.-,trust_env%20(bool)%20%E2%80%93,-Trust%20environment%20settings):
```
trust_env ([bool](https://docs.python.org/3/library/functions.html#bool)) –

Trust environment settings for proxy configuration if the parameter is True (False by default). See [Proxy support](https://docs.aiohttp.org/en/stable/client_advanced.html#aiohttp-client-proxy-support) for more information.

Get proxy credentials from ~/.netrc file if present.

Get HTTP Basic Auth credentials from ~/.netrc file if present.

If [NETRC](https://docs.aiohttp.org/en/stable/glossary.html#envvar-NETRC) environment variable is set, read from file specified there rather than from ~/.netrc.
```

Reply:
You can inject by adding `session: Inject[aiohttp.ClientSession]` to  its `__init__`  and assigning it to a prop there.

The [proxy](https://github.com/RedHatInsights/astro-virtual-assistant-v2/blob/main/docs/dev/proxy.md) is already used from the [HTTPS_PROXY env](https://github.com/RedHatInsights/astro-virtual-assistant-v2/blob/main/services/watson-extension/.env.template#L5) 

i.e. `HTTPS_PROXY=http://squid.corp.redhat.com:3128`

Reply:
Yeah, I am not sure either. 😅 
I am not able to repro the issue from our pods, so I was trying to throw anything at it

Reply:
@josejulio  Ah, I just updated this PR by setting `ssl=False` for the request to status.redhat.com.
I think that could fix the issue from the pods:
```
Cannot connect to host status.redhat.com:443 ssl:True
```